### ISSUE TITLE
LGR Images starting with "q" are not added to the list of pictures/textures/masks

### ISSUE THREAD

**Issue by Smibu** [Smibu/elmanager#116]
In GitLab by @sunlelma on Aug 20, 2017, 23:49

It looks like to screen whether lgr images should be in the selectable dropdown list you use
```
                    if (listedImage.Name[0] != 'q')
                    {
                        add image to list
                    }
```

An image named "quack" or "queen" will not show up in the list of images from which to select (i.e. picture/mask/texture)

The proper code should be as follows:

```

special_list=[
    "q1body", "q1thigh", "q1leg", "q1bike", "q1wheel", "q1susp1", "q1susp2",
    "q1forarm", "q1up_arm", "q1head", "q2body", "q2thigh", "q2leg", "q2bike",
    "q2wheel", "q2susp1", "q2susp2", "q2forarm", "q2up_arm", "q2head", "qflag",
"qkiller", "qexit", "qframe", "qcolors",
"qfood1", "qfood2", "qfood3", "qfood4",
"qfood5", "qfood6", "qfood7", "qfood8", "qfood9","qgrass"]


if listedImage.Name is not in special_list
and
listedImage.Name[:4] != "qup_"
and
listedIImage.Name[:6] != "qdown_"
{
add image to list
}
```

.lower() if necessary

and obviously you already are handling the special case of  "qgrass" so I included it in the above list

**Comment by Smibu** [comment]
In GitLab by @sunlelma on Aug 20, 2017, 23:49

changed the description

**Comment by Smibu** [comment]
In GitLab by @sunlelma on Aug 20, 2017, 23:50

changed the description

**Comment by Smibu** [comment]
In GitLab by @sunlelma on Aug 21, 2017, 00:08

changed the description

**Comment by Smibu** [comment]
In GitLab by @sunlelma on Aug 21, 2017, 00:08

changed the description

**Comment by Smibu** [comment]
In GitLab by @Smibu on Sep 9, 2017, 20:30

Ok thanks. Should be easy to fix.

Do you have some example LGR that I could use to test this?

**Comment by Smibu** [comment]
In GitLab by @sunlelma on Sep 10, 2017, 00:48

[qhas.lgr](/uploads/0fdd81c4e469c5e1de52815875c8a49f/qhas.lgr)

Added the following files to lgr:

```
qsusp_1;susp.bmp;PICTURE;380;U;TOPLEFT
qqsusp_2;susp.bmp;PICTURE;380;U;TOPLEFT
QQ_3;susp.bmp;PICTURE;380;U;TOPLEFT
Qxsa_4;susp.bmp;PICTURE;380;U;TOPLEFT
QQds_5;susp.bmp;PICTURE;380;U;TOPLEFT
uQQQ_6;susp.bmp;PICTURE;380;U;TOPLEFT
Q1hi_7;susp.bmp;PICTURE;380;U;TOPLEFT
Qflage_8;susp.bmp;PICTURE;380;U;TOPLEFT
Qfreme_9;susp.bmp;PICTURE;380;U;TOPLEFT
Q2leg_0;susp.bmp;PICTURE;380;U;TOPLEFT
qmaskbig;maskbig.bmp;MASK;0;U;TOPLEFT
qstone1;stone1.bmp;TEXTURE;750;G;TOPLEFT
```

**Comment by Smibu** [comment]
In GitLab by @Smibu on Dec 3, 2017, 20:47

closed via commit 2e77d90ce4c4ca57ffe6d5c1eae528161d328950
